_docs/geyser/anticheat-compatibility.md

```diff
@@ -10,16 +10,11 @@ title: Anticheat Compatibility
 
 Full Compatibility (Checks Bedrock Players Accurately)
 
-- [AntiAura](https://www.spigotmc.org/resources/1368/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
```

Comment:
Is there an official statement that this is done or just assuming since there's been no updates?

Reply:
The last update is from January last year. The dev was still active 3 month ago on his discord. 
I think we should not remove this one if it's not officially unsupported.